assets/css/_logged_in_as.scss

```diff
@@ -0,0 +1,32 @@
+.c-logged-in-as__container {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
+
+.c-logged-in-as__icon {
+  background-color: #6d39ac;
```

Comment:
**question:** if I'm not mistaken this is the new eggplant-500 that we're not actually using yet, so I guess there's a question of which to use for now. I'm asking design in the context of my own ticket FWIW.

Reply:
Oh heh true.

Reply:
👋 https://github.com/mbta/skate/pull/2383/commits/8fb447e8adc48f1158038538c65f40796cb520eb